core/common/src/main/java/alluxio/Constants.java

```diff
@@ -97,6 +99,7 @@
   public static final long LINEAGE_MASTER_WORKER_SERVICE_VERSION = 1;
   public static final long KEY_VALUE_MASTER_CLIENT_SERVICE_VERSION = 1;
   public static final long KEY_VALUE_WORKER_SERVICE_VERSION = 1;
+  public static final long FILE_SYSTEM_CLIENT_SERVICE_VERSION = 1;
```

Comment:
we do not distinguish between the protocol type for other version constants ... for instance, BLOCK_MASTER_CLIENT_SERVICE_VERSION is used both in the Thrift block master client and REST block master client ... the service this version is for is "File System"